src/connection/socket-connection.ts

```diff
@@ -183,10 +183,8 @@ export class SocketConnection {
    * Wraps the incoming `message` in a `JSON RPC Success Response` using the original subscription`JSON RPC Id` to send through the WebSocket.
    */
   private createSubscriptionHandler(id: JsonRpcId): (message: GenericMessage) => void {
-    return (message) => {
-      const response = createJsonRpcSuccessResponse(id, { reply: {
-        record : message
-      } });
+    return (event) => {
+      const response = createJsonRpcSuccessResponse(id, { reply: { event } });
```

Comment:
For my understanding: What's the important point of changing this to "event"? Is it so that it's general thus applicable to both record and event subscriptions?

Reply:
Slightly modified to remove the redundant reply wrapper. 

The "event" represents the `MessageEvent` type instead of just the message as it was before we included the initialWrite with the events we are emitting.